samples/CAPI/CMakeLists.txt

```diff
@@ -1,12 +1,12 @@
 find_package(lua 5.3 REQUIRED)
 
-set(CAPI_SOURCES
+set(SOURCES
```

Comment:
So I'm curious - what's with the shared `SOURCES` CMake variable and others that you seem to re-use?

Reply:
Each CMakeLists is ran in its own scope; this is just simplifying the names where having a prefix was unnecessary

Reply:
Fair enough!

Reply:
This is what @FiniteReality called out on the other PR (https://github.com/novelrt/NovelRT/pull/278#discussion_r602920760). 

> a lot of names could likely be simplified, as the CMake set() command only operates within the current scope (unless PARENT_SCOPE is specified), and each CMakeLists is in its own scope IIRC.

This means we can just keep it simple as `SOURCES` and it won't conflict across the distinct CMakeLists.txt